app/models/miq_server/server_timezone.rb

```diff
@@ -0,0 +1,44 @@
+class MiqServer
+  module ServerTimezone
+    extend ActiveSupport::Concern
+
+    # return timezone abbreviation
+    def timezone_abbr(user = nil)
+      if user.nil?
+        tz = server_timezone
+        time = Time.now
+      else
+        tz = Time.zone
+        time = Time.zone.now
+      end
+      time.in_time_zone(tz).strftime("%Z")
+    end
+
+    # returns utc_offset of timezone
+    def timezone_offset(user = nil, formatted = false)
+      tz = user_timezone(user)
+      tz = ActiveSupport::TimeZone::MAPPING[tz]
+      ActiveSupport::TimeZone.all.each do |a|
+        if ActiveSupport::TimeZone::MAPPING[a.name] == tz
+          if formatted
+            return a.formatted_offset
+          else
+            return a.utc_offset
+          end
+        end
+      end
+    end
+
+    # most of our tests are setup to not have my_server
+    # so probably want to use the followign instead:
+    # user ? user.get_timezone : server_timezone
```

Comment:
yea, I'm going to delete that method and move the tests to user / server_timezone.

using the `user_timezone` method instead of the longer version blew up too many tests since we don't always have a MiqServer.my_server